developer/src/kmc-keyboard-info/src/index.ts

```diff
@@ -210,20 +198,8 @@ export class KeyboardInfoCompiler {
     }
     keyboard_info.packageIncludes = [...includes];
 
-    // Always overwrite source data
-    if(sources.kmpFileName) {
-      this.setField(keyboard_info, 'version', sources.kmpJsonData.info.version.description);
-    } else {
-      const m = jsFile?.match(/this\.KBVER\s*=\s*(['"])([^'"]+)(\1)/);
```

Comment:
Do we not need to fallback to KBVER anymore?

Reply:
I verified that all keyboards in the repo have a package -- and going forward we won't accept keyboards that _don't_ have a package, so the version info is always readable from there.

There are still keyboards that don't have a .kmx, and those ones need us to parse the .js for other fields such as `isRTL`, so we still load the .js for those.